ui/src/components/Header.tsx

```diff
@@ -78,86 +81,79 @@ export default function DashboardNavbar({
           </div>
           <div className="flex w-full items-center justify-end gap-x-2">
             <div className="flex shrink-0 items-center space-x-4">
-              {showConnectionStatus && (
-                <div className="hidden items-center gap-x-2 md:flex">
-                  <div className="w-[159px]">
-                    <PeerConnectionStatusCard
-                      state={peerConnectionState}
-                      title={kvmName}
-                    />
-                  </div>
-                  <div className="hidden w-[159px] md:block">
-                    <USBStateStatus
-                      state={usbState}
+              <div className="hidden items-center gap-x-2 md:flex">
```

Comment:
First big change here is putting the three right-side things in a single row so the heights line up.